docs/build/operator-class.ipynb

```diff
@@ -0,0 +1,745 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b88d9242-a6f2-47b6-8e0a-daf60d0c4374",
+   "metadata": {},
+   "source": [
+    "# The Operator class\n",
+    "\n",
+    "This page shows how to use the [`Operator`](/api/qiskit/qiskit.quantum_info.Operator) class. For a high-level overview of the class, see [Overview of operator classes](/build/operators-overview#operator)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "549bc166-9c0e-4e89-b7e2-8adafd7110c3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from qiskit.circuit import QuantumCircuit\n",
+    "from qiskit.circuit.library import CXGate, RXGate, XGate\n",
+    "from qiskit.quantum_info import Operator, Pauli, process_fidelity\n",
+    "from qiskit_aer import Aer"
```

Comment:
Why are we using Aer in this notebook? I think we should swap out for real hardware

Reply:
This material is simply copied over from the other notebook. Changing the material should be done as a separate issue.

Reply:
To address your actual comment though, I think it's appropriate to use a simulator on this page because the content is theoretical.

Reply:
Ok, I went and actually tried to understand why Aer was being used. I couldn't make sense of it and it seems like that code cell was completely unrelated to the rest of the content, so I deleted it.